0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>A small thing that I could perhaps see happening is that step 2 is interpreted as running both cells below, which can mess up the rest of the notebook. </p><p> </p><p>I might move this block after defining <code>parameters_list</code>, or explicitly state that they should only run the cell where  <code>parameters_list</code> is defined once. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='126'/>

Reply:
<p>Thanks so much ! This will def be fixed! </p>